docs/bank-connect/react.md

```diff
@@ -49,4 +49,21 @@ Please make sure from date is always less than to date.
 :::
 
 ## Callback
+
+`onSuccess` function will return a payload which will contain information from which further data can be extracted. The payload structure of `onSuccess` is as follows:
+
+```js
+const onSuccess = (payload) => {
+    console.log("Payload", payload)
+}
+```
+`payload` will have following structure.
+
+```json
```

Comment:
Can you check if the keys are correct below, also should the keys be in double-quotes for valid json?

Reply:
ive changed the json. and keys has been fixed